content/v2.0/reference/api/influxdb-1x/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: InfluxDB 1.x compatibility API
+description: >
+  placeholder
+menu:
+  v2_0_ref:
+    name: 1.x compatibility
+    parent: InfluxDB v2 API
+weight: 104
+v2.0/tags: [influxql, query, write]
+products: [cloud]
+related:
+  - /v2.0/query-data/influxql
+---
+
+The InfluxDB v2 API includes InfluxDB 1.x compatibility endpoints that work with
+InfluxDB 1.x client libraries and third-party integrations like [Grafana](https://grafana.com) and others.
+
+## Authentication
+InfluxDB 2.0 requires all query and write requests to be authenticated.
+Use **basic authentication** or **token authentication** to authenticate requests to
+InfluxDB 1.x compatibility endpoints.
+
+### Basic Authentication
+Basic authentications requires the following credentials:
+
+- **username**: InfluxDB username
```

Comment:
is this the influxdb 2.0 username which is the owner of the token?  do we know if this is actually validated?
meaning could you put any random string of characters here along with `:token` and that would work?

Reply:
Tokens don't belong to users, they belong to orgs. It does check to see if the username exists in the org associated with the provided token. If the username doesn't exist, access is denied. I've tested this and it works.

Reply:
I can verify this behavior is codified in the source